taskcat/template_params.py

```diff
@@ -0,0 +1,361 @@
+import re
+import random
+import uuid
+import logging
+from taskcat.colored_console import PrintMsg
+from taskcat.exceptions import TaskCatException
+
+logger = logging.getLogger('taskcat')
+class ParamGen(object):
```

Comment:
we no longer have to subclass object in python.
```suggestion
 class ParamGen:
```